curriculum/challenges/english/25-front-end-development/lab-moon-orbit/66a37f37ef5823a313de8c26.md

```diff
@@ -0,0 +1,362 @@
+---
+id: 66a37f37ef5823a313de8c26
+title: Build a Moon Orbit
+challengeType: 14
+dashedName: build-a-moon-orbit
+demoType: onClick
+---
+
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
```

Comment:
I wouldn't say that. Solar system refers to the Sun and the bodies orbiting around it.

What about this?
```md
In this lab, you will create a simple animation of the Moon's orbit around the Earth using HTML and CSS. The Earth will be at the center of the system, and the Moon will orbit around it.
```

Reply:
Yeah, that fine. As long as we provide some so campers will not what they are getting into.

IMO, most of these labs should have descriptions because we want to make sure that campers understand what they are building.